src/theme/components/socialmedialinks.ts

```diff
@@ -0,0 +1,168 @@
+// maybe set the border to always be on, but invisible if prop is set to "none"?
+
+import { StyleFunctionProps } from "@chakra-ui/theme-tools";
+
+function getRadius(size) {
+  let radiusSize = "18px";
+  switch (size) {
+    case "medium":
+      radiusSize = "24px";
+      break;
+    case "large":
+      radiusSize = "32px";
+  }
+  return {
+    radiusSize,
+  };
+}
+
+function getColor(color) {
+  let ltColor = "ui.typography.heading",
+    dkColor = "dark.ui.typography.heading",
+    ltBgColor,
+    dkBgColor;
+
+  switch (color) {
+    case "link":
+      ltColor = "ui.link.primary";
+      dkColor = "dark.ui.link.primary";
+      break;
+    case "textInverse":
+      ltColor = "ui.typography.inverse.heading";
+      dkColor = "dark.ui.typography.inverse.heading";
+      ltBgColor = "ui.typography.heading";
+      dkBgColor = "dark.ui.typography.heading";
+  }
+
+  return {
+    ltColor,
+    dkColor,
+    ltBgColor,
+    dkBgColor,
+  };
+}
+
+const SocialMediaLinks = {
+  // The parts of the component
+  parts: ["li", "a", "svg", "span"],
```

Comment:
FYI, this is not wrong but it's not needed. This is used for custom names when using the `useMultiStyleConfig` style hook. Then you can do something like `styles.customContainer`. Taking a closer look, you're using `useMultiStyleConfig` now which is not wrong but because this component is simple and you only use the direct `style` object, you can switch with `useStyleConfig` on line 128 of `SocialMediaLinks.tsx`.

Reply:
When I switch to UseStyleConfig, the textInverse color is not visible b/c the <ul> has the same background color as the inverse. I am unsure how to manipulate the <ul> styles w/o UseMultiStyleConfig.

Reply:
Okay, keep it for now. Once your PR is ready for review I can take a closer look.

Reply:
It seems that using `useStyleConfig` seems to be okay now? Can you give it another try?

Reply:
Yes, it does work now. Why???